### PR TITLE
Improve .env file and brunch instructions

### PR DESCRIPTION
**Pull request by craiglyons**
This addresses issues people may have with installing brunch and getting their `.env` file created properly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,9 @@ You'll need these dependencies for Razor & your new Phoenix app:
 * [Elixir 1.4.5](https://elixir-lang.org/install.html)
 * Postgres
 * [Yarn](https://github.com/yarnpkg/yarn) for JavaScript dependencies, installed and available on your path.
-* [Brunch](https://github.com/brunch/brunch) for JavaScript builds, installed and available on your path.
+* [Brunch](https://github.com/brunch/brunch) for JavaScript builds, installed and available on your path.  
+You can install brunch globally or local to your project with yarn or npm.  
+If you install it local to your project, you'll need to add `node_modules` to your path so the executable can be found.
```
Comment (bunnymatic) [inline README.md]: should read

```you'll need to add `node_modules/.bin` to your path so```

File: README.md [hunk 2]
```diff
@@ -42,8 +44,10 @@ _Note for [asdf](https://github.com/asdf-vm/asdf) users - both Razor & the gener
 * `cd your_new_project_dir`
 * `mix deps.get` - install dependencies
 * `mix ecto.setup` - setup your local database
-* Prep js - `yarn install; brunch build`
-* Create your .env with a secret key base, i.e. for bash: `echo -n 'export SECRET_KEY_BASE=' > .env; mix phoenix.gen.secret >> .env`
+* Prep js - `yarn install; brunch build`
```
Comment (bunnymatic) [inline README.md]: This could be `yarn install && brunch build`

File: README.md [hunk 3]
```diff
@@ -42,8 +44,10 @@ _Note for [asdf](https://github.com/asdf-vm/asdf) users - both Razor & the gener
 * `cd your_new_project_dir`
 * `mix deps.get` - install dependencies
 * `mix ecto.setup` - setup your local database
-* Prep js - `yarn install; brunch build`
-* Create your .env with a secret key base, i.e. for bash: `echo -n 'export SECRET_KEY_BASE=' > .env; mix phoenix.gen.secret >> .env`
+* Prep js - `yarn install; brunch build`  
+If you have issues with this, see the "Before You Start" section above.
+* Create a `.env` file with variables MIX_ENV=dev and SECRET_KEY_BASE=your_secret_key_base.  
+Run `mix phoenix.gen.secret` to generate a good value for `SECRET_KEY_BASE`.
```
Comment (bunnymatic) [inline README.md]: 👍

File: lib/presenter.ex [hunk 4]
```diff
@@ -35,16 +35,16 @@ defmodule Razor.Presenter do
...
+    # For a local copy, install brunch from here and add node_modules to your path
     $ brunch build
 
-    # Create your .env with a secret key base, i.e. for bash:
-    $ echo -n 'export SECRET_KEY_BASE=' > .env; mix phoenix.gen.secret >> .env
-    # Set dev mode locally
-    $ echo -n 'export MIX_ENV=dev' >> .env
+    # Create a .env file with variables MIX_ENV=dev and SECRET_KEY_BASE=your_secret_key_base
+    # To generate a good SECRET_KEY_BASE, you can use the phoenix generator:
+    $ mix phoenix.gen.secret
```
Comment (bunnymatic) [inline lib/presenter.ex]: i like it.

File: README.md (hunk 1 above)
Comment (craiglyons) [inline README.md]: Good catch! Shall fix.

File: README.md (hunk 2 above)
Comment (craiglyons) [inline README.md]: Fish shell doesn't have &&, so I opted for the portable one-liner.

**Pull request merged**